spec/octopus/serverspec/json/api/smtpconfiguration.json

```diff
@@ -0,0 +1,18 @@
+{
+
+  "Id": "smtp",
+  "SmtpHost": "smtp.fictionaldomain.com",
```

Comment:
```suggestion
  "SmtpHost": "smtp.example.com",
```

Reply:
example.com etc is setup deliberately for this kind of example.

If you change these, the tests will need to change too